modules/sharedaddy/sharing.css

```diff
@@ -753,3 +753,14 @@ div.sharedaddy.sharedaddy-dark #sharing_email {
 	height: 123px;
 	margin: 0 0 1em 0;
 }
+
+.sd-content .share-customize-link {
+	margin-top: -.7em; 
+	line-height: 11px;
+}
+
+.sd-content .share-customize-link a{
+	font-size: 11px !important;
```

Comment:
Yeah, I tend to avoid it as well; in this case the issue requirements mentioned matching the style used on the "{x} bloggers liked this" text on the Likes widget; I copied these styles from there, where we were also using `!important`, so I assumed the intention is to make sure that we are overriding any theme link styles. Not sure though.

Reply:
Looks like we're adding this markup? Overall, it feels okay either way, if other themes were building on top of this class, we'd probably require the `!important`

Reply:
🤔 overall I'm not sure how much we care about the font size sticking to 11px vs scaling with the theme. For example, notice how large the Edit control is here:

<img width="661" alt="Screen Shot 2020-07-15 at 10 25 46 AM" src="https://user-images.githubusercontent.com/1270189/87576080-b3efab00-c685-11ea-8249-648e82ab9049.png">

Reply:
Since the spirit of the styling comes from the "(x bloggers) like this" text below Likes, I think pinning it is reasonable (that Like text is pinned to the same values).